person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_writer :name, :age
```

Comment:
- [ ] According to the project requirement, `name` and `age` instance variables should be able to be modified. That is why it will be nice they are defined as `attr_accessors` (here both `Getters` and `Setters` should be defined)